src/screens/Feed/FeedScreen.tsx

```diff
@@ -25,23 +25,40 @@ const FeedScreen = ({ navigation }: any) => {
   return (
     <View style={styles.container}>
       <View style={styles.feedHeader}>
-        <Text style={styles.feedTitle}>CNSC Job Postings</Text>
+        <View style={styles.redSquare}></View>
+        <Text style={styles.feedTitle}>Welcome!</Text>
+      </View>
+      <ScrollView
+        contentContainerStyle={{
+          flexGrow: 1,
+          alignItems: 'center',
+          width: '100%'
+        }}>
+        {list.map((job) => {
```

Comment:
It seems like the way the feed is currently styled means that only ~7 job cards are visible when there are 16 jobs in firebase. This doesn't can be addressed in a new PR but you should make a new sprint task for it and then resolve it (should take priority over other tasks for now).